Makefile

```diff
@@ -269,9 +269,6 @@ build-collector build-collector-debug:
 build-ingester build-ingester-debug:
 	$(GOBUILD) $(DISABLE_OPTIMIZATIONS) -o ./cmd/ingester/ingester$(SUFFIX)-$(GOOS)-$(GOARCH) $(BUILD_INFO) ./cmd/ingester/main.go
 
-.PHONY: docker
```

Comment:
I still find the docker and docker-push targets useful. I had to use them during the last release, as I had to manually build/publish the images.

Reply:
I see, now you can use `scripts/build-upload-docker-images.sh` to build/publish multi-arch images.